pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/somatic/result/SomaticResultMetaData.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.dsm.model.somatic.result;
+
+import lombok.NonNull;
+
+public class SomaticResultMetaData {
```

Comment:
Can you add a comment saying what this class is for. (It is not yet obvious from a review perspective, but there may be a pattern I am missing.)